e2e/student/studentCanOpenProfessorsProfile.spec.ts

```diff
@@ -49,6 +57,8 @@ test.describe('Student can search professor and open professors profile', async
   test('Search not existing text', async ({page}, testInfo) => {
     const searchingOfProfessor: SearchingOfProfessor = {
       requestText: 'ბლაბლაბლა',
+      professorsEmail: null,
+      role: null,
       positive: false,
```

Comment:
need to remove , since in testMethod if (searchingOfProfessor.positive)  will be work always